packages/main/src/plugin/kubernetes/context-permissions-checker.ts

```diff
@@ -58,7 +58,7 @@ export interface ContextPermissionResult extends Permission {
 }
 
 export class ContextPermissionsChecker implements Disposable {
-  #contextName: string;
+  contextName: string;
```

Comment:
this is a problem there as it's a public attribute there

I think it should stay with #contextName (private)


and then add a `public get contextName(): string {` method that will expose the context but only as a read only attribute